### PR TITLE
Toolbar displaced issues 1357 1412

### PR DESCRIPTION
**Pull request by AlbertoCasasOrtiz**
Fixes issues #1357 #1412

### Brief summary of changes

Merged changes from previous PRs #1377 and #1414

### Testing I've completed

 - [x] Windows
 - [x] Mac
 - [x] Linux

### CHANGELOG.md (choose one)

- Updated.

### CODE REVIEW

**Comment by AlbertoCasasOrtiz** [comment]
I have tested it on windows, mac and ubuntu and looks like everything is working properly.

**Comment by aymanhab** [comment]
@AlbertoCasasOrtiz For some reason I can't download the artifacts from github, is that an issue on my side or a temporary problem?

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,8 +7,9 @@ request related to the change, then we may provide the commit.
 v4.4.1
 ======
 - Fix issue [#1378](https://github.com/opensim-org/opensim-gui/issues/1378): In Static Optimization Tool, dialog box has wrong title for "Directory" in output section.
-- Fix issue [#1395](https://github.com/opensim-org/opensim-gui/issues/1395) where visual selection and properties windows are not in sync.
-
+- Fix issue [#1395](https://github.com/opensim-org/opensim-gui/issues/1395): Where visual selection and properties windows are not in sync.
+- Fix issue [#1357](https://github.com/opensim-org/opensim-gui/issues/1357): Toolbar displaced and non visible after resizing window
+- Fix issue [#1412](https://github.com/opensim-org/opensim-gui/issues/1412): Missing Tools in Tools Menu.
```
Comment (aymanhab) [inline CHANGELOG.md:13]: I'd remove #1412 from change log since it was a side effect of an earlier commit that no users have been exposed to.

**Review by aymanhab — APPROVED** [review]
Other than the CHANGELOG.md changes LGTM

**Pull request merged**